src/main/java/mate/academy/bookstore/validation/FieldMatchValidator.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.bookstore.validation;
+
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import org.apache.commons.beanutils.BeanUtils;
+
+public class FieldMatchValidator implements ConstraintValidator<FieldMatch, Object> {
+    private String firstFieldName;
+    private String secondFieldName;
+
+    @Override
+    public void initialize(FieldMatch constraintAnnotation) {
+        this.firstFieldName = constraintAnnotation.first();
+        this.secondFieldName = constraintAnnotation.second();
+    }
+
+    @Override
+    public boolean isValid(Object value, ConstraintValidatorContext context) {
+        try {
+            Object firstValue = BeanUtils.getProperty(value, firstFieldName);
```

Comment:
Or use can use Spring BeanWrapperImpl and remove org.apache.commons.beanutils dependency
`Object firstValue = new BeanWrapperImpl(value).getPropertyValue(this. firstFieldName);`